drivers/include/tmp006.h

```diff
@@ -84,8 +84,8 @@ extern "C"
 {
 #endif
 
-#ifndef TMP006_I2C_ADDRESS
-#define TMP006_I2C_ADDRESS         0x41 /**< TMP006 Sensor Default Address */
+#ifndef TMP006_ADDR
+#define TMP006_ADDR                0x40 /**< TMP006 Sensor Default Address */
```

Comment:
The board.h itself defines the address and thus the address pin setting. On phyNODEs one pin is high and one is low which leads to I2C address 0x41 (IIRC). However, using e.g. an Atmel board with the tmp006 adafruit sensor breakout board I prefer to define the "default" as "not connected". IIRC the sensor breakout board has pull-down resistors on it for the address lines.